po/zh-Hans.po

```diff
@@ -3054,24 +3054,28 @@ msgstr ""
 
 #: src/exercises/day-1/for-loops.md:1
 msgid "# Arrays and `for` Loops"
-msgstr ""
+msgstr "# 数组与 `for` 循环"
 
 #: src/exercises/day-1/for-loops.md:3
 msgid "We saw that an array can be declared like this:"
-msgstr ""
+msgstr "我们可以这样声明一个数组："
 
 #: src/exercises/day-1/for-loops.md:5
 msgid ""
 "```rust\n"
 "let array = [10, 20, 30];\n"
 "```"
 msgstr ""
+"```rust\n"
+"let array = [10, 20, 30];\n"
+"```"
 
 #: src/exercises/day-1/for-loops.md:9
 msgid ""
 "You can print such an array by asking for its debug representation with `{:?}"
 "`:"
 msgstr ""
+"要输出上面声明的数组，可以使用 `{:?}` 来请求它的调试表示："
```

Comment:
The following suggestion perhaps don't convey the meaning of asking for, but I thinking it is easier to understand for Chinese reader and it doesn't change the main meaning of this sentence.
```suggestion
"你可以使用`{:?}` 来打印这种数组的调试格式："
```

Reply:
@sylxjtu Please recheck this modification when you have time，I wonder if it is more understandable for readers. If there is no problem with this I'll approve this pr and merge it.